timApp/static/scripts/tim/user/settings.component.ts

```diff
@@ -435,7 +435,12 @@ type StyleDocumentInfoAll = Required<StyleDocumentInfo>;
                         </div>
                     </ng-container>
                     <ng-container>
-                        <span class="form-label" i18n>Machine translator API Keys</span>
+                        <span class="form-label" i18n>Machine translator API Keys
```

Comment:
Se nyt taitaa napata myös koko URLin mukaan käännökseen. 

Tuossa voisi kokeilla laittaa

```html
<ng-container i18n>Machine translator API Keys</ng-container>
``` 

ja ottaa `<span>`sta `i18n` pois. Näin se sitten ottaa vain tuon tekstin käännökseen, mutta ei koko spanin sisällön.

Reply:
Jes, mä kokeilen tuota.